packages/mobile/src/screens/chat-screen/ChatListItemSkeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import { View } from 'react-native'
+
+import Skeleton from 'app/components/skeleton'
+import { makeStyles } from 'app/styles'
+
+import { useStyles as useChatListItemStyles } from './ChatListItem'
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  profilePicture: {
+    width: spacing(12),
+    height: spacing(12),
+    borderRadius: spacing(6)
+  },
+  userNameContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: spacing(3)
+  },
+  userName: {
+    height: spacing(4),
+    width: spacing(25),
+    borderRadius: spacing(12)
+  },
+  handle: {
+    height: spacing(3),
+    width: spacing(22),
+    borderRadius: spacing(12)
+  },
+  latestMessage: {
+    marginTop: spacing(2),
+    height: spacing(4),
+    width: spacing(57),
+    borderRadius: spacing(12)
+  }
+}))
+
+export const ChatListItemSkeleton = ({
+  shouldFade = false,
+  index = 0
+}: {
+  shouldFade?: boolean
+  index?: number
+}) => {
+  const styles = useStyles()
+  const ChatListItemStyles = useChatListItemStyles()
+
+  return (
+    <View
+      style={[
+        ChatListItemStyles.root,
+        shouldFade ? { opacity: (4 - index) * 0.25 } : null
```

Comment:
Might need a comment on why you are doing this or a helper function to isolate this special case logic